modules/spring-config/src/main/java/org/mule/runtime/config/spring/SpringXmlConfigurationBuilder.java

```diff
@@ -79,7 +81,15 @@ else if (useDefaultConfigResource)
             allResources.add(new ConfigResource(MULE_REGISTRY_BOOTSTRAP_SPRING_CONFIG));
             allResources.add(new ConfigResource(MULE_MINIMAL_SPRING_CONFIG));
             allResources.add(new ConfigResource(MULE_SPRING_CONFIG));
-            allResources.add( new ConfigResource(MULE_DEFAULTS_CONFIG));
+            allResources.add(new ConfigResource(MULE_DEFAULTS_CONFIG));
+            try
+            {
+                allResources.add(new ConfigResource(MULE_DEFAULTS_ENDPOINT_CONFIG));
```

Comment:
We need to add a build that verifies that when the transport module is in the classpath, even if the transports are not used, all the test cases run successfully.